src/extensions/averagewaveforms/AverageWaveformsView/AverageWaveformView.tsx

```diff
@@ -31,6 +31,7 @@ const AverageWaveformView: FunctionComponent<Props> = ({ boxSize, plotData, args
     return (
         <WaveformWidget
             waveform={plotData.average_waveform}
+            layoutMode={selection.waveformsMode || 'geom'}
```

Comment:
layoutMode tells whether to use elec geom to display the waveforms. These screenshots demonstrate the two alternatives:

![image](https://user-images.githubusercontent.com/3679296/104366965-1ff18580-54e8-11eb-8cca-2751649c0234.png)

![image](https://user-images.githubusercontent.com/3679296/104367010-2ed83800-54e8-11eb-8498-993cfd407dff.png)